iam.tf

```diff
@@ -70,10 +70,7 @@ resource "aws_iam_role_policy" "codebuild_role_policy" {
       ],
       "Condition": {
         "StringEquals": {
-          "ec2:Subnet": [
-            "${data.aws_subnet.private_subnet_a.arn}",
-            "${data.aws_subnet.private_subnet_b.arn}"
-          ],
+          "ec2:Subnet": data.aws_subnet.private_subnets.*.arn,
```

Comment:
I think there's a slightly more preferred splat expression for this in TF12: https://www.terraform.io/docs/configuration/expressions.html#splat-expressions

e.g. private_subnets[*].arn but not sure on the exact syntax.